### DISCUSSION TITLE
MTG/packs/snc-02

### DISCUSSION THREAD

**Discussion by giscus** [Nerban/MTG#8] [BoosterComments]
# MTG/packs/snc-02

https://nerban.github.io/MTG/packs/snc-02

**Comment by Nerban** [comment]
Here is my take:

### Theme of the (Un)Common

I get the vibe of a low life criminal lifestyle in the streets of New Capenna.
With Tavern Swindler, Psychic Pickpocket and Prizefight and Capenna Express it's the day-to-day life while strangle Exotic Pets and Caldaia Strongarm the deadly aspect of that lifestyle.


### Favorite

My Favorite with this pack has to be Ob Nixilis, the adversary scince it provides a synergy with my favorite Card (Ashe Reaper) so these two will end up in a Deck together at some point.

## What do you think?
- whats your Take on the Theme?
- what are your thoughts on playability, synergy or design?
